frontend/src/api/ClientsRepository.js

```diff
@@ -22,4 +22,20 @@ export default {
       )
     );
   },
+
+  assignClientAliases(clients) {
+    let clientAliases = [
```

Comment:
Looks like this is just using an array as a map, so it could probably just be that instead which would simplify things and might be more performant (map `.get()` vs. array `.find()`). That being said, it's a small array and likely not going to grow much past this, so this is more of an "FYI" comment instead of a "do this or I will not approve" comment.